rollbar/__init__.py

```diff
@@ -702,6 +719,12 @@ def _report_exc_info(exc_info, request, extra_data, payload_data, level=None):
     if extra_trace_data and not extra_data:
         data['custom'] = extra_trace_data
 
+    if scopes:
+        data['scope'] = scopes[-1]
```

Comment:
A couple of questions, why we set only last scope?

Reply:
Right. So, we only send the last scope in the stack of `scopes` because that is the most current scope and the assumption/intention is that this is the only one that is relevant. This also comes from our LaunchDarkly integration where we can only edit one feature flag at a time and that is dependent on one scope.

Reply:
@ajtran I think scopes is something that can be merged all together when be sent, as we do in the Ruby library. Different sections can have different scopes but I think we'd like to keep all the inner and outer scopes